src/core/networkreply.h

```diff
@@ -178,7 +178,7 @@ class NetworkReply : public QObject
     /**
      * Number of retries left. Once the value reaches zero, the status of the last reply is the final status.
      */
-    int mRetriesLeft = 5;
+    int mRetriesLeft = 0;
```

Comment:
@suricactus , are you planning to re-enable this before merging the PR?

Reply:
Not in this PR, it was causing more pain than solutions. The problem is that the payload in POST requests is being lost between the retries. I have an idea why, but it would be faster to just merge like this and debug later.